kedro-tutorial/src/kedro_tutorial/pipeline.py

```diff
@@ -72,8 +71,18 @@ def create_pipeline(**kwargs):
     """
     de_pipeline = Pipeline(
         [
-            node(preprocess_companies, "companies", "preprocessed_companies"),
-            node(preprocess_shuttles, "shuttles", "preprocessed_shuttles"),
+            node(
+                preprocess_companies,
+                "companies",
+                "preprocessed_companies",
+                name="preprocess1",
+            ),
+            node(
+                preprocess_shuttles,
+                "shuttles",
+                "preprocessed_shuttles",
+                name="preprocess2",
+            ),
```

Comment:
Yes, because `params:` prefix would be useful if you want to only use one  or few of parameters in `parameters.yml`, whereas in this case we are using all parameters (two in this case) in `parameters.yml`, so it is actually easy to use `parameters` rather than `params:`